### ISSUE TITLE
tty not supported

### ISSUE THREAD

**Issue by achimgaedke** [achimgaedke/py-aws-vault-auth#2]
```
aws-vault: error: exec: Failed to get credentials for XXX: open /dev/tty: device not configured
```

aws-vault has started to use tty instead of stdin/stdout: https://github.com/99designs/aws-vault/blob/master/prompt/terminal.go with commit https://github.com/99designs/aws-vault/commit/e7d4a173f8e80dc179378bd62e8e29edd16b3c3c / version https://github.com/99designs/aws-vault/releases/tag/v7.1.0

**Comment by achimgaedke** [comment]
The first version of pty-based communication is in branch https://github.com/achimgaedke/py-aws-vault-auth/tree/pty-experiments

**Comment by achimgaedke** [comment]
fixed with [37f26c5](https://github.com/achimgaedke/py-aws-vault-auth/commit/37f26c5478a237756bb0809a236f258daebc1560)
